src/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableQueue_1.cs

```diff
@@ -151,7 +154,7 @@ public ImmutableQueue<T> Enqueue(T value)
 
             if (this.IsEmpty)
             {
-                return new ImmutableQueue<T>(ImmutableStack<T>.Empty.Push(value), ImmutableStack<T>.Empty);
+                return new ImmutableQueue<T>(ImmutableStack.Create(value), ImmutableStack<T>.Empty);
```

Comment:
How is `ImmutableStack.Create(value)` better than `ImmutableStack<T>.Empty.Push(value)`?